lib/command/src/Obelisk/Command/Project.hs

```diff
@@ -137,12 +138,29 @@ findProjectObeliskCommand target = do
         ]
       return Nothing
 
+-- We need the umask not be too strict with the group permissions on the folders
+-- Obelisk is using, since some systems have a default umask of 0002 instead of
+-- 0022, and we want to respect that. Since we have to set the umask to read it,
```

Comment:
Please put this warning as part of the docs for `getDefaultUmask`.

Reply:
Done